pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -532,6 +532,42 @@ func timeoutFunc(d time.Duration) time.Duration {
 // GetAvailableNotifiers returns the metadata of all the notification channels that can be configured.
 func (am *Alertmanager) GetAvailableNotifiers() []*alerting.NotifierPlugin {
 	return []*alerting.NotifierPlugin{
+		{
```

Comment:
I think it's about time we move this to a different file, moreso does this have to come from the *Alertmanager? My preference would be that this be attached to the *API directly or part of that package.

Reply:
> moreso does this have to come from the *Alertmanager

Nope. Good point. We were trying to avoid global variables. But an exported function looks better here.